scss/_utilities/_clearfix.scss

```diff
@@ -0,0 +1,19 @@
+// ==========================================================================
+// Clearfix
+// ==========================================================================
+
+
+@mixin clearfix() {
+	&:before,
+	&:after {
+		content: "";
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+}
+
+.clearfix {
```

Comment:
Regarding naming here this really is dependent upon final decision on https://github.com/jquery/css-chassis/issues/4